lib/maze/option.rb

```diff
@@ -3,6 +3,11 @@
 # Provides the set of Maze Runner command line options
 module Maze
   module Option
+    # Document server options (ds = document server)
+    DS_BIND_ADDRESS = 'ds-bind-address'
+    DS_PORT = 'ds-port'
+    DS_ROOT = 'ds-root'
```

Comment:
I think if the name needs to be explained in a comment it could be clearer. `--document-server-bind-address` is a bit long but given this will be scripted almost all the time maybe that's ok?

Perhaps merging the bind address and port would let this be `--document-server`? Ruby's URI module can turn that back into a host/port pair:

```ruby
require "uri"

address = "localhost:8080" # from --document-server
# otherwise URI doesn't know what kind of URI it is
address = "http://#{address}" unless address.start_with?("http")

uri = URI(address)
port = uri.port # => 8080
host = uri.host # => "localhost"
```

We'd still need `--document-server-root` but that's not too much longer than `--ds-bind-address`

Reply:
Combining them into `--document-server` seems like a nice approach, but would we want it to be inconsistent with the existing `--bind-address` and `--port` options?  And what if someone wants to change the port without giving a specific bind address (which I believe we had problems with before)?

Reply:
How about `--doc-server-bind-address` as some kind of compromise?

Reply:
I think that still works so long as you always prefix the port with `:`, e.g. `--document-server :1234`:

```ruby
irb(main):001:0> a = URI('http://:1234')
irb(main):002:0> a.port
=> 1234
irb(main):003:0> a.host
=> nil
```

Reply:
I think that still works so long as you always prefix the port with `:`, e.g. `--document-server :1234`:

```ruby
irb(main):001:0> a = URI('http://:1234')
irb(main):002:0> a.port
=> 1234
irb(main):003:0> a.host
=> nil
```

`--doc-server-` is better than `--ds-` but I think I still prefer `--document-server-`. What about calling it a file server instead? `--file-server-` is only 1 char longer than `--doc-server-` but is a bit clearer IMO

Reply:
Agreed to go with `--document-server...`, as we'd typically expect the config to be set in code anyway.